gem/lib/chargebee_rails/webhook_handler.rb

```diff
@@ -130,33 +134,33 @@ def existing_payment_method
       @existing_payment_method ||= ::PaymentMethod.where(cb_customer_id: event.content.customer.id)
     end
 
-    def can_sync? obj
+    def can_sync?(obj)
       obj.first && (obj.first.event_last_modified_at.to_i < event.occurred_at)
     end
 
-    def subscription_attrs subscription
+    def subscription_attrs(subscription)
       {
         chargebee_id: subscription.id,
         plan_id: ::Plan.find_by(plan_id: subscription.plan_id).id,
         plan_quantity: subscription.plan_quantity,
         status: subscription.status,
-        event_last_modified_at: event.occurred_at,
+        event_last_modified_at: event.occurred_at ? Time.at(event.occurred_at) : Time.now,
```

Comment:
@ghiculescu can you tell us why we need the conditional statement here ? Is there any scenario where event.occurred_at can be nil ?

Reply:
i don't remember anymore :( i think i am just being defensive.

Reply:
:sweat_smile: I will merge PR now and later update that statement when we are confident of it. Confident Ruby !